### DISCUSSION TITLE
Dna.Json

### DISCUSSION THREAD

**Discussion by Majakhtar** [nftchef/art-engine#80] [Help + Q&A] [answered]
Hey,

I wanted to know what the relevance of the dna.json file is please, also is this file supposed to contain the DNA sequence that is shown in the individual json files, as my folder just shows: (I have inserted Filename below instead of the name of my actual files name)
"0.0:Filename#20.png*1.0:Filename#100.png*2.5:Filename#30.png*3.2:Filename#20.png*4.3:Filename#5.png*5.4:Filename#30.png*6.5:Filename#10.png",

Thank you

**Comment by nftchef** [comment, marked as answer]
The _dna.json file is an output of the DNA struct used during generation. 
It is used in a couple of different util scripts, for example, continuing generation without any duplicates from the first run, or rebuilding the entire collection (either without the background layer or different resolution or corrected art file, etc.)

and yes, it will have a reference to the files used in the image and the attributes selected, using the file names.
